### PR TITLE
Add color_extractor service

### PR DESCRIPTION
**Pull request by GenericStudent**
## Proposed change

This PR links up with the new ColorThief component. It allows service call to pass in either a URL or a file path to an image file, colorthief then gives us the predominant color (RGB tuple) which we then set to the referenced light entity.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/42129
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by GenericStudent** [comment]
Just a note, I have a Ubuntu 20 enviornment and Centos 7 environment. Neither are working using the default instructions to install ruby and build docs :( so have been unable to view the MD locally.

Edit: I had version 0.36 in my fork and didn't update 🤦‍♂️
Now rebased with latest next branch and I have it previewing locally 🎉

File: source/_integrations/color_extractor.markdown [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: Color Extractor
+description: Instructions how to integrate the Color Extractor into Home Assistant.
+logo: home-assistant.png
```
Comment (klaasnicolaas) [inline source/_integrations/color_extractor.markdown]:
```suggestion
```
Please remove this line

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/color_extractor.markdown [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      data_template:
+        color_extract_url: '{{ states.media_player.chromecast.attributes.entity_picture }}'
+        entity_id: light.shelf_leds
+```
+
+With a nicer transition period of 5 seconds and setting brightness to 100% each time (part of the [`light.turn_on`](/integrations/light#service-lightturn_on) service parameters):
+
+```yaml
+#automation.yaml
+- alias:
```
Comment (klaasnicolaas) [inline source/_integrations/color_extractor.markdown]:
```suggestion
- alias:
```

Missing a nice name 😉

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/color_extractor.markdown (hunk 2 above)
Comment (GenericStudent) [reply source/_integrations/color_extractor.markdown]: Good catch 😀

**Review by frenck — APPROVED** [review]
Thanks, @GenericStudent 👍

**Pull request merged**